### PR TITLE
Create debts page

### PR DESCRIPTION
**Pull request by Boburm1rzo**
Before changes

image

Last changes
image

### CODE REVIEW

File: CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
+﻿using CheckDrive.Web.Stores.Debts;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckDrive.Web.Controllers
+{
+    public class DebtsController : Controller
```
Comment (Ramadan1011) [inline CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs:6]: should be better to be semicolon at the end of namespace
image

Comment (Ramadan1011) [reply CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs:6]: like this

File: CheckDrive.Web/CheckDrive.Web/Views/Debts/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                <e-grid-column headerText="Bor edi" field="FuelAmount" textAlign=Center headerTextAlign=Center width="40"></e-grid-column>
+                <e-grid-column headerText="To'langan" field="PaidAmount" textAlign=Center headerTextAlign=Center width="50"></e-grid-column>
+                <e-grid-column headerText="Status" template="#statusColumnTemplate" textAlign=Center headerTextAlign=Center width="80"></e-grid-column>
+                <e-grid-column width="180" type="text" textAlign="Center" template="#actionTemplate"></e-grid-column>
+            </e-grid-columns>
+        </ejs-grid>
+    </div>
+</div>
+
+
```
Comment (jamshidbek7375) [inline CheckDrive.Web/CheckDrive.Web/Views/Debts/Index.cshtml:65]: Do not leave a space

**Review by jamshidbek7375 — APPROVED** [review]

File: CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        private readonly IDebtsStore _store;
+        public DebtsController(IDebtsStore debtsStore)
+        {
+            _store = debtsStore;
```
Comment (Ramadan1011) [inline CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs:11]: should we check that is debtsStore not null and if debtsStore is null, return new ArgumentNullException

File: CheckDrive.Web/CheckDrive.Web/wwwroot/js/site.js [hunk 4]
```diff
@@ -28,4 +28,12 @@
         }
     }
 });
+function toggleDropdown() {
```
Comment (Ramadan1011) [inline CheckDrive.Web/CheckDrive.Web/wwwroot/js/site.js:31]: separate with an empty line

**Review by Ramadan1011 — CHANGES_REQUESTED** [review]

File: CheckDrive.Web/CheckDrive.Web/Views/Debts/Index.cshtml (hunk 2 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/Views/Debts/Index.cshtml:65]: fixed

File: CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs (hunk 1 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs:6]: fixed

File: CheckDrive.Web/CheckDrive.Web/wwwroot/js/site.js (hunk 4 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/wwwroot/js/site.js:31]: fixed

File: CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs (hunk 3 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/Controllers/DebtsController.cs:11]: fixed

**Review by diyorzz — APPROVED** [review]

**Review by Ramadan1011 — APPROVED** [review]
Looks good

**Review by jamshidbek7375 — APPROVED** [review]

**Review by FirdavsAX — APPROVED** [review]

**Review by khojianvar — APPROVED** [review]

**Pull request merged**